packages/devtools/src/commands/findElement.js

```diff
@@ -8,9 +8,17 @@ export default async function findElement ({ using, value }) {
         throw new Error(`selector strategy "${using}" is not yet supported`)
     }
 
-    const page = this.windows.get(this.currentWindowHandle)
+    const page = this.getPageHandle()
     let needsCleanUp = false
 
+    if (using === 'link text') {
+        using = 'xpath'
+        value = `.//a[normalize-space() = "${value}"]`
```

Comment:
why do we need `.` here? There should be no nested element when using findElement(s), I think.